README.md

```diff
@@ -57,6 +57,24 @@ If you do not wish to compile anything, simply download the file under [Releases
 
 If you are using an Arch-based distribution, install [`mangohud`](https://aur.archlinux.org/packages/mangohud/) and [`lib32-mangohud`](https://aur.archlinux.org/packages/lib32-mangohud/) with your favourite AUR helper. [`mangohud-git`](https://aur.archlinux.org/packages/mangohud-git/) and [`lib32-mangohud-git`](https://aur.archlinux.org/packages/lib32-mangohud-git/) are also available on the AUR if you want the up-to-date version of MangoHud.
 
+If you are building it by yourself, you need to enable multilib repository, by editing pacman config:
+
+```
+sudo nano /etc/pacman.conf
```

Comment:
*cries in vim*

Jokes aside, while we all know what it means, maybe avoid using specific ways to gain root privilages in readme similar to how Arch Wiki does it? I'd have to see how the instructions are in other projects. :thinking: